README.md

```diff
@@ -39,6 +39,9 @@ stored as a singleton.
 Interested? Then have a look at our
 [quick start](https://stockwatchdev.github.io/application_settings/develop/docs/Quick_start/).
```

Comment:
Maybe we should add a link here as well to the full documentation. So something like:
Then have a look at our [quick start](https://stockwatchdev.github.io/application_settings/develop/docs/Quick_start/) or the [full documentation](https://stockwatchdev.github.io/application_settings/develop/)